### PR TITLE
Make a number of minor grammar/syntax/font/NOTE fixes to the first pa…

### PR DESCRIPTION
**Pull request by rpjday**
…rt of Chapter 7.

Changes include:

  - standardizing commands and filenames in monospaced font
  - italics for emphasis
  - add NOTE explaining double "-f" option for "git clean"

### CODE REVIEW

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 1]
```diff
@@ -277,3 +277,10 @@ What now>
 ----
 
 This way you can step through each file individually or specify patterns for deletion interactively.
+
+[NOTE]
+====
+There is a quirky situation where you might need to be extra forceful in asking Git to clean your working directory. 
+It's possible that you have a working directory wherein one or more subdirectories are _themselves_ Git clones, which means you have subdirectories that contain their own `.git` directories.
+Even with the "force" `-f` option, Git will not remove ignored subdirectories that contain either a file or directory named `.git` -- to get rid of those, you need to use _two_ `-f` options for emphasis.
+====
```
Comment (ben) [inline book/07-git-tools/sections/stashing-cleaning.asc]: 🤔 The way you're describing this isn't really a supported use case in Git. What do you think about this?
```
If you happen to clone or copy a Git repository into the working directory of another one, even `git clean -fd` will refuse to delete it.
In this situation, you have to provide a second `-f` flag to get rid of it.
```

Comment (rpjday) [reply book/07-git-tools/sections/stashing-cleaning.asc]: OK, I'll deal with that this evening.

Comment (rpjday) [reply book/07-git-tools/sections/stashing-cleaning.asc]: OK, I compromised, the new wording is somewhere in between.

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 2]
```diff
@@ -281,6 +281,6 @@ This way you can step through each file individually or specify patterns for del
 [NOTE]
 ====
 There is a quirky situation where you might need to be extra forceful in asking Git to clean your working directory. 
-It's possible that you have a working directory wherein one or more subdirectories are _themselves_ Git clones, which means you have subdirectories that contain their own `.git` directories.
-Even with the "force" `-f` option, Git will not remove ignored subdirectories that contain either a file or directory named `.git` -- to get rid of those, you need to use _two_ `-f` options for emphasis.
+If you happen to be in a working directory under which you've copied or cloned other Git repositories (perhaps as submodules), even `git clean -fd` will refuse to delete those directories.
```
Comment (ben) [inline book/07-git-tools/sections/stashing-cleaning.asc]: Hmm, I hate bringing submodules into it like this, but I guess that's probably the primary use case for having two `-f` flags.

**Pull request merged**